components/Tabs/src/TabBarView/private/MDCTabBarViewItemView.m

```diff
@@ -122,6 +131,17 @@ - (void)layoutSubviews {
   }
 }
 
+- (CGRect)contentFrameForTitleLabelInTextOnlyLayout {
+  CGRect contentFrame = UIEdgeInsetsInsetRect(self.bounds, kEdgeInsetsTextOnly);
+
+  CGSize contentSize = CGSizeMake(CGRectGetWidth(contentFrame), CGRectGetHeight(contentFrame));
+  CGSize labelWidthFitSize = [self.titleLabel sizeThatFits:contentSize];
+  CGSize labelSize =
+      CGSizeMake(labelWidthFitSize.width, MIN(contentSize.height, labelWidthFitSize.height));
+  return CGRectMake(CGRectGetMidX(contentFrame) - (labelWidthFitSize.width / 2),
+                    CGRectGetMidY(contentFrame), labelSize.width, labelSize.height);
```

Comment:
Why are we using the MidY and not the MinY?

Reply:
Oops, good catch. This should be `midY - labelSize.height / 2`

Reply:
Fixed and added tests to make sure the contentFrame makes sense.